### PR TITLE
[APLO] New Object Type Application Log Objects

### PR DESCRIPTION
**Pull request by wurzka**

### CODE REVIEW

File: file-formats/aplo/type/zif_aff_aplo_v1.intf.abap [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "! <p class="shorttext">APLO Object Type</p>
+    "! Object type APLO
+    BEGIN OF ty_main,
+      "! $required
+      format_version TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header         TYPE ty_header,
+      "! <p class="shorttext">Array of Subobjects</p>
```
Comment (wurzka) [inline file-formats/aplo/type/zif_aff_aplo_v1.intf.abap:46]:
```suggestion
      "! <p class="shorttext">Subobjects</p>
```

File: file-formats/aplo/type/zif_aff_aplo_v1.intf.abap [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      "! $values {@link zif_aff_types_v1.data:co_abap_language_version_cloud}
+      "! $default {@link zif_aff_types_v1.data:co_abap_language_version_cloud.standard}
+      abap_language_version TYPE zif_aff_types_v1=>ty_abap_language_version,
+    END OF ty_header.
+
+  TYPES:
+    "! <p class="shorttext">Subobject</p>
+    "! Properties of a subobject
+    BEGIN OF ty_subobject,
+      "! <p class="shorttext">Subobject Name</p>
```
Comment (wurzka) [inline file-formats/aplo/type/zif_aff_aplo_v1.intf.abap:27]:
```suggestion
      "! <p class="shorttext">Name</p>
```

File: file-formats/aplo/type/zif_aff_aplo_v1.intf.abap [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      "! Name of the subobject
+      "! $required
+      name        TYPE c LENGTH 20,
+      "! <p class="shorttext">Subobject Description</p>
```
Comment (wurzka) [inline file-formats/aplo/type/zif_aff_aplo_v1.intf.abap:31]:
```suggestion
      "! <p class="shorttext">Description</p>
```

File: file-formats/aplo/type/zif_aff_aplo_v1.intf.abap [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  TYPES:
+    "! <p class="shorttext">APLO Object Type</p>
+    "! Object type APLO
+    BEGIN OF ty_main,
+      "! $required
+      format_version TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header         TYPE ty_header,
```
Comment (wurzka) [inline file-formats/aplo/type/zif_aff_aplo_v1.intf.abap:45]:
```suggestion
      header         TYPE zif_aff_types_v1=>ty_header_60_cloud,
```

File: file-formats/aplo/type/zif_aff_aplo_v1.intf.abap [hunk 5]
```diff
@@ -0,0 +1,50 @@
+INTERFACE zif_aff_aplo_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Header for APLO Object</p>
+    "! The header for an application log object
+    BEGIN OF ty_header,
```
Comment (wurzka) [inline file-formats/aplo/type/zif_aff_aplo_v1.intf.abap:7]: Can be removed (see ty_main)

**Review by burcuka — APPROVED** [review]

**Pull request merged**